src/test/java/com/spotify/futures/CompletableFuturesTest.java

```diff
@@ -331,7 +360,7 @@ public void dereference_exceptional() throws Exception {
 
   @Test
   public void dereference_null() throws Exception {
-    final CompletionStage<Object> dereferenced = dereference(completedFuture(null));
+    final CompletionStage<Object> dereferenced = dereference(completedFutureFrom(null));
```

Comment:
this test should be duplicated, not replaced

Reply:
This was a mistake when renaming - will fix!